docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Hue 4.10 and its new SQL Editor component, REST API, small File Importer and Slack App are out!
+author: Hue Team
+type: post
+date: 2021-06-14T00:00:00+00:00
+url: /blog/hue-4-10-sql-scratchpad-component-rest-api-small-file-importer-slack-app/
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.10
+  - Release
+
+---
+Hi Data Explorers,
+
+The Hue Team is glad to thanks all the contributors and release Hue 4.10!
+
+<a href="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png">
+  <img src="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png" />
+</a>
+
+&nbsp;
+
+Here is a summary of the [main improvements](/categories/version-4.10/) of 4.10 on top of the previous [4.9](/blog/hue-4-9-sql-dialects-phoenix-dasksql-flink-components/) release:
+
+
+## SQL Editor Component & API
+Now build your own SQL Editor in two HTML line by leveragin the [SQL Scratchpad](https://docs.gethue.com/developer/components/scratchpad/) component. The first version of the public [REST API](https://docs.gethue.com/developer/api/rest/) for executing queries was published to.
```

Comment:
"line by levragin" => "lines by leveraging" and "queries was published to" => "queries was published too"

Reply:
"two HTML line" we say three below.